yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I think this is a file containing metadata that is not necessary for the master branch. I think we should remove it like the other metadata files. :)